MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/APICall.cs

```diff
@@ -36,6 +37,10 @@ public List<Photo> GetMarsRoverPhotosByDateAndRover(DateTime earth_date, string
             }
             request.AddQueryParameter("api_key", _apiKey);
             var response = _restClient.Get<MarsRoverPhotosResponse>(request);
+            if (response.StatusCode != System.Net.HttpStatusCode.OK)
+            {
+                throw new Exception("API Call invalid");
```

Comment:
Probably worth finding a more specific type of Exception for this. Also is this message ever going to be displayed to the user? If so it should probably be changed to be a bit less technical.